specification/dimensions/commitmentdiscountname.md

```diff
@@ -0,0 +1,32 @@
+# Commitment Discount Name
+
+A commitment-based discount is a contractual commitment for an amount of usage or spend, for a specified term, in exchange for a discounted unit price. Commitment-based discounts can cover either usage or cost of various resources such as virtual machines or databases. A provider can offer various pricing models, payment options, and term durations for commitment-based discounts.
+
+A Commitment Discount Name is the display name assigned to a commitment-based discount.
+
+The CommitmentDiscountName column MUST be present in the billing data. This column must be of type String. The CommitmentDiscountName value MUST be null if the charge is not related to a commitment-based discount and MAY be null if a display name cannot be assigned to a commitment-based discount. CommitmentDiscountName MUST NOT be null if a display name can be assigned to a commitment-based discount.
```

Comment:
Same comments as for CommitmentDiscountId:

> when a charge is related to a commitment-based discount
Maybe: 'when a record is related to a commit-based discount'.

Indeed, in case of a resource covered by commitment-based discount, the charge (BilledCost) would be 0.
Maybe it could be explictly mentioned: the CommitDiscountId value must not be null if the record relates to the purchase of a commitment-based discount or if the record relates to the usage of a resource covered by a commitment-based discount.

Reply:
As per Udam's comment in CommitDiscountId PR, the word 'charge' in FOCUS refers to record / line item / row.